database/factories/AddressFactory.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Faker\Generator as Faker;
+use GetCandy\Api\Core\Addresses\Models\Address;
+
+/*
+|--------------------------------------------------------------------------
+| Model Factories
+|--------------------------------------------------------------------------
+|
+| This directory should contain each of the model factory definitions for
+| your application. Factories provide a convenient way to generate new
+| model instances for testing / seeding your application's database.
+|
+*/
+
+$factory->define(Address::class, function (Faker $faker) {
+    return [
+        'salutation' => $faker->title,
+        'firstname' => $faker->firstName,
+        'lastname' => $faker->lastName,
+        'email' => $faker->safeEmail,
+        'phone' => $faker->phoneNumber,
+        'company_name' => $faker->company,
+        'address' => $faker->streetAddress,
+        'address_two' => $faker->streetName,
+        'address_three' => null,
+        'city' => $faker->city,
+        'county' => $faker->state,
```

Comment:
I'm not sure how well used "county" is as a name, we may wish to change to "state", I'm not sure.

Reply:
Yeah it's a tricky one, I did look at Fedex (wasn't that helpful) and also looked at Amazon but then I guess with Amazon it's country specific. I.e. US users would see state, cause I had County on my account...

Reply:
Faker uses State and quite a few other countries have States. BUT counties are normally below state level, e.g. a USA state will then have counties below it. We're not interested in those however.

Do we need a state/county field, well yes for things like SagePay. So I think we go with "state".

Reply:
Okay so then it's just up to the front end to display the correct label I guess?

Next question is what happens to the data in the county column for users that are upgrading

Reply:
I guess if the state field is empty, shove it in there... otherwise shove it in address_three, and if that's not free, append it to the state field?? We probably don't need to be too concerned, as it's still in beta.